Signal/src/Models/OWSCall.h

```diff
@@ -15,9 +15,9 @@ typedef enum : NSUInteger {
     kCallMissed = 3,
     kCallOutgoingIncomplete = 4,
     kCallIncomingIncomplete = 5,
-    // kGroupUpdateJoin has been deprecated.
-    kGroupUpdateLeft = 7,
-    kGroupUpdate = 8,
+    // kGroupUpdateJoin = 6, has been deprecated.
+    // kGroupUpdateLeft = 7, has been deprecated.
+    // kGroupUpdate = 8, has been deprecated.
```

Comment:
I'm OK either way - I'm interpreting your last comment to mean that you're OK with the current implementation. If not LMK and I can open a new PR.